### ISSUE TITLE
GOREL fixes to stop using GOREL

### ISSUE THREAD

**Issue by pgaudet** [geneontology/go-ontology#19497]
* any entity without a GOREL ID, only a RO ID, will be completely removed from GOREL 
(unless they are used a a parent of another class)
* The goal is to obsolete all terms: once they are not in use anymore in P2GO and MGI annotation extensions, and in the ontology

**Comment by pgaudet** [comment]
@cmungall Is this plan OK with you ? 
Are you aware of other groups than GO using GOREL that we would need to notify ? 

Thanks, Pascale

**Comment by vanaukenk** [comment]
@pgaudet @cmungall @balhoff 

We discussed gorel on the go-cam specifications call today.

One question we had was if gorel.obo is, or can be, generated from gorel-edit.owl.  The gorel ids are properly represented in gorel-edit.owl, so if that file was the source for gorel.obo, the id issue could potentially be solved?

Also, I didn't think the plan was to actually remove any relations from gorel, but rather to obsolete the relations we will no longer use in annotation or the ontology.

**Comment by balhoff** [comment]
@vanaukenk yes gorel.obo is generated from gorel-edit.owl. So really the RO terms don't need to be manually removed from there (they are automatically pulled in—@pgaudet I forgot this earlier when we were discussing this).

I'm not sure if that is what you mean by the "id issue". I think what @pgaudet has planned is consistent with what you're saying: work towards obsoleting all gorel terms.

**Comment by vanaukenk** [comment]
Thanks @balhoff 

The "id issue" I was referring to is that in gorel.obo, the GOREL:nnnnnnn ids are in the xref field rather than the id field.

Here's an example using 'activated by'

[Typedef]
**id: activated_by**
name: activated_by
namespace: go/extensions/gorel
comment: This relation should not be used to annotate one gene product regulating another's activity; a regulator MF or regulation BP should be used in such cases.
subset: AE_chemical
subset: display_for_curators
subset: valid_for_annotation_extension
synonym: "activated by" EXACT RO:0008000 [PomBase:curators]
synonym: "activated_by_chemical" EXACT [GOC:mah]
synonym: "activated_by_substance" EXACT [GOC:mah]
synonym: "activity_increased_by" EXACT [GOC:mah]
**xref: GOREL:0000507**

**Comment by balhoff** [comment]
@vanaukenk that's a normal OBO format thing. It's a convenience so that you see `activated_by` throughout the file when you look at term relations. You will see the same thing in `go-edit.obo`. Protege automatically uses the xref for the IRI when it parses the file.

**Comment by vanaukenk** [comment]
Okay, I didn't know that about the OBO format, and I hadn't noticed it in go-edit.obo before.

This whole issue came up in the context of producing a GPAD for import from annotations in Protein2GO and how to get the ids for obsolete gorel relations.  I think there may actually be two separate issues, now, but this at least explains what we see in gorel.obo

Thank you for the explanation!
